packages/t/trompeloeil/xmake.lua

```diff
@@ -0,0 +1,70 @@
+package("trompeloeil")
+
+    set_homepage("https://github.com/rollbear/trompeloeil")
+    set_description("A thread-safe header-only mocking framework for C++11/14 using the Boost Software License 1.0")
+    set_license("BSL-1.0")
+
+    add_urls("https://github.com/rollbear/trompeloeil/archive/refs/tags/$(version).tar.gz")
+    add_versions("v43", "86a0afa2e97347202a0a883ab43da78c1d4bfff0d6cb93205cfc433d0d9eb9eb")
+
+    on_install(function (package)
+        os.cp("include/*", package:installdir("include"))
+    end)
+
+    on_test(function (package)
```

Comment:
too long test, can you simplify it?

Reply:
Yeah, I thought that might have been. No problem.